gui/wxpython/image2target/ii2t_gis_set.py

```diff
@@ -824,8 +824,8 @@ def DeleteMapset(self, event):
                 shutil.rmtree(os.path.join(self.gisdbase, location, mapset))
                 self.OnSelectLocation(None)
                 self.lbmapsets.SetSelection(0)
-            except:
-                wx.MessageBox(message=_("Unable to delete mapset"))
+            except (OSError, shutil.Error) as e:
```

Comment:
It doesn't look like shutil.Error is raised:
```
>>> try:
...     shutil.rmtree("/tmp/aaaa/aaa")
... except shutil.Error as e:
...     print(e)
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "/usr/lib/python3.12/shutil.py", line 775, in rmtree
    onexc(os.lstat, path, err)
  File "/usr/lib/python3.12/shutil.py", line 773, in rmtree
    orig_st = os.lstat(path, dir_fd=dir_fd)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/aaaa/aaa'
```

Reply:
I'll add `FileNotFoundError` as well, which I think should cover pretty much all major errors `shutil.rmtree` can raise. Please let me know if you think a catch all `Exception` would be a better choice here

Reply:
Got it, I'll only keep OSError in that case